manifests/candlepin.pp

```diff
@@ -105,8 +105,7 @@
       target => $keystore,
       owner  => 'tomcat',
       group  => $group,
-    }
-
+    } ->
```

Comment:
Does this require the entire stack above it to execute or do they just need the password file to exist per the usage in the `import CA` step?